alexawp.php

```diff
@@ -139,6 +140,18 @@ public function add_image_size() {
 		add_image_size( 'alexa-small', 720, 480, true );
 		add_image_size( 'alexa-large', 1200, 800, true );
 	}
+
+	/**
+	 * Add the Alexa service to the list of allowed HTTP origins
+	 *
+	 * @param $allowed_origins array Default allowed HTTP origins
+	 * @return array allowed origin URLs
+	 */
+	public function allowed_http_origins( $allowed_origins ) {
+		$allowed_origins[] = 'http://ask-ifr-download.s3.amazonaws.com';
```

Comment:
The documentation only stated the http url, but adding couldn't hurt :) https://developer.amazon.com/blogs/post/Tx15T24P9QV8RXN/New-ASK-Features-Standard-Home-Cards-with-Image-Support